src/components/common/journey/journey.ts

```diff
@@ -0,0 +1,52 @@
+import { JOURNEY_TYPE } from "../../../app.constants";
+import { UserSession } from "../../../types";
+
+type GetJourneyTypeFromUserSessionOptions = {
+  fallbackJourneyType?: JOURNEY_TYPE;
+  includeReauthentication?: true;
+  includeAccountRecovery?: true;
+};
+
```

Comment:
These journey files will be removed from this PR once #1290 merges and this PR is rebased